arbitrum-docs/node-running/how-tos/running-an-orbit-node.mdx

```diff
@@ -1,85 +1,127 @@
 ---
 title: 'How to run a full node for an Orbit chain'
-description: Learn how to run an Arbitrum orbit node on your local machine.
-author: greatsoshiant
-sme: greatsoshiant
+description: 'Learn how to run an Arbitrum orbit node on your local machine'
+author: Mehdi Salehi
+sme: Mehdi Salehi
 content_type: how-to
 ---
 
 This how-to provides step-by-step instructions for running an Orbit node on your local machine.
 
-## Minimum hardware configuration
+### Minimum Hardware Configuration
```

Comment:
Shouldn't titles be sentence-case? (or did this change?)
i.e. `Minimum hardware configuration`

Reply:
You're right. That's fixed.